packages/hebao_v1/contracts/test/PriceCacheStore.sol

```diff
@@ -77,14 +74,14 @@ contract PriceCacheStore is DataStore, PriceOracle, OwnerManagable
         uint    value
         )
         external
-        onlyManager
+        onlyOwnerOrManager
     {
         cacheTokenPrice(token, amount, value);
     }
 
     function setOracle(PriceOracle _oracle)
         external
-        onlyManager
+        onlyOwnerOrManager
```

Comment:
I still prefer manager-only to force the owner to add and use a manager so that the owner address will not be used in any hot wallet at all.

Reply:
Okay, will revert.